### ISSUE TITLE
heading angle missing in HR SLC patches

### ISSUE THREAD

**Issue by agrouaze** [umr-lops/xsar_slc#65]

**Comment by rmarquarlops** [comment]
Only the `platform_heading` is accessible in the attributes of the generated patches.

**Comment by alxmouche** [comment]
ground heading is different from "platform heading". We are after "ground heading". This information should also be included in the L1B on the 3 grids (intra, inter and HR) for L2 application

**Comment by alxmouche** [comment]
A mon avis à mettre dans la v1.5

**Comment by rmarquarlops** [comment]
Computing the heading angle is done this way in xsar:   
   
```
    def coords2heading(self, lines, samples, to_grid=False, approx=True):
        """
        Get image heading (lines increasing direction) at coords `lines`, `samples`.
        Parameters
        ----------
        lines: np.array or scalar
        samples: np.array or scalar
        to_grid: bool
            If True, `lines` and `samples` must be 1D arrays. The results will be 2D array of shape (lines.size, samples.size).
        Returns
        -------
        np.array or float
            `heading` , with shape depending on `to_grid` keyword.
        """

        lon1, lat1 = self.coords2ll(lines - 1, samples, to_grid=to_grid, approx=approx)
        lon2, lat2 = self.coords2ll(lines + 1, samples, to_grid=to_grid, approx=approx)
        _, heading = haversine(lon1, lat1, lon2, lat2)
        return heading
```

with coords2ll converting (line, sample) coordinates of a given grid pixel to (lon, lat) coordinates.
So it converts a (line, sample) coordinates on both sides of the considered point in the line axis and applies the haversine function to obtain the ground heading angle.

**Comment by lanougue** [comment]
I used tile's corner aligned in line direction to compute the heading
solved in #81
